uncontrast_studies/tests/test_distribution_of_effects_across_parameters.py

```diff
@@ -0,0 +1,135 @@
+from rest_framework import status
+
+from contrast_api.choices import StudyTypeChoices, PresentationModeChoices, SignificanceChoices, UnConSampleChoices
+from uncontrast_studies.tests.base import UnContrastBaseTestCase
+
+
+class TestDistributionOfEffectsAcrossParametersGraphTestCase(UnContrastBaseTestCase):
+    def _setup_world(self):
+        study = self.given_study_exists(
+            title="Israeli study",
+            countries=["IL"],
+            DOI="10.1016/j.cortex.2017.07.011",
+            year=2002,
+            type=StudyTypeChoices.UNCONSCIOUSNESS,
+        )
+        main_paradigm = self.given_uncon_main_paradigm_exists("main_paradigm")
+        specific_paradigm = self.given_uncon_specific_paradigm_exists("specific_paradigm", main=main_paradigm)
+        stimulus_modality_type = self.given_uncon_stimulus_modality_type_exists("modality_type")
+        stimulus_category_type = self.given_uncon_stimulus_category_type_exists("category_type")
+        unconsciousness_measure_phase_type = self.given_unconsciousness_measure_phase_exists("phase_type")
```

Comment:
naming "phase_type" is confusing and seems unnecessary to me

Reply:
agreed would fix